src/main/java/com/inventor/email/Email.java

```diff
@@ -0,0 +1,46 @@
+package com.inventor.email;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Getter;
+import lombok.Setter;
+import java.io.Serializable;
+import java.util.Date;
+
+@Getter
+@Setter
+public class Email implements Serializable {
+    private String oderzhuvach;
+    private String tema;
+    private String tekst;
+
+    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="dd.MM.yyyy HH:mm", timezone="EET")
+    private Date dataOtrymannya;
```

Comment:
java.util.Date is a legacy class which should be avoided whenever possible. Please consider using newer Java Time API which is located in java.time package. Also, back-end should not bother with date formatting, that's the task of the client. Java Time classes are serialized to standard formats with which clients can work and display them in any format and timezone they need